### PR TITLE
last_updated_at() function completed

### PR DESCRIPTION
**Pull request by codesleep-Beperfect**

### CODE REVIEW

File: src/scrape_up/github/respository.py [hunk 1]
```diff
@@ -351,6 +354,19 @@ def get_contributors(self):
...
+
+    def last_update_at(self):
+        data=self.__scrape_page()
+        try:
+            update=data.find_all("relative-time", class_="no-wrap")
+            return update[0].get_text()
+        except:
+            message="Oops! No Repo or Organization found"
+            return message
+
```
Comment (nikhil25803) [inline src/scrape_up/github/respository.py:366]: Do follow the return type convention of the project, and check other methods for reference.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

File: src/scrape_up/github/respository.py (hunk 1 above)
Comment (codesleep-Beperfect) [reply src/scrape_up/github/respository.py:369]: I have done it from my end. Now, you can check it and merge it.

File: src/scrape_up/github/respository.py [hunk 2]
```diff
@@ -351,6 +354,22 @@ def get_contributors(self):
                 "message": message,
             }
 
+
+    def last_update_at(self):
+        data=self.__scrape_page()
+        try:
+            update=data.find_all("relative-time", class_="no-wrap")
+            return update[0].get_text()
```
Comment (nikhil25803) [inline src/scrape_up/github/respository.py:362]: Follow the return type format in try block also.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

File: src/scrape_up/github/respository.py (hunk 2 above)
Comment (codesleep-Beperfect) [reply src/scrape_up/github/respository.py]: Done.

**Pull request merged**